### PR TITLE
Fix OSX SDK pinning for qt-main on arm64

### PR DESCRIPTION
**Pull request by andfoy**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/gen_patch_json.py [hunk 1]
```diff
@@ -283,6 +283,8 @@
     'openmpi-4.0.1-hc99cbb1_1': '10.12',
     'openmpi-4.0.1-hc99cbb1_0': '10.12',
     'openmpi-4.0.1-hc99cbb1_2': '10.12',
+    'qt-main-5.15.2-hfc65906_0': '11.0',
+    'qt-main-5.15.2-h76a31b5_1': '11.0',
```
Comment (isuruf) [inline recipe/gen_patch_json.py:287]: This sets the minimum to be 11.0, but we want to remove the minimum.

File: recipe/gen_patch_json.py [hunk 2]
```diff
@@ -1485,6 +1490,20 @@ def _set_osx_virt_min(fn, record, min_vers):
         record['constrains'] = run_constrained
 
 
+def _fix_qt_main(fn, record):
```
Comment (isuruf) [inline recipe/gen_patch_json.py:1493]:
```suggestion
def _remove_osx_virtual_dep(fn, record):
```

**Comment by andfoy** [comment]
@isuruf, is this one ready?

**Comment by hmaarrfk** [comment]
Sorry for letting this go. I guess people forget to check up on all PRs. is this still relevant today?

**Comment by andfoy** [comment]
This was before updating Qt to 5.15.14, so I think this can be marked as not needed anymore?

**Pull request closed without merging**